### PR TITLE
[MIG] [15.0] account_invoice_margin

### PR DESCRIPTION
**Pull request by nedaszilinskas**

### CODE REVIEW

**Review by legalsylvain — APPROVED** [review]
Diff review. LGTM. Thanks for porting this module.

File: account_invoice_margin/tests/test_account_invoice_margin.py [hunk 1]
```diff
@@ -120,7 +120,13 @@ def test_invoice_refund(self):
                 active_ids=self.invoice.ids,
                 active_id=self.invoice.id,
             )
-            .create({"refund_method": "refund", "reason": "reason test create"})
+            .create(
+                {
+                    "refund_method": "refund",
+                    "reason": "reason test create".format,
```
Comment (pedrobaeza) [inline account_invoice_margin/tests/test_account_invoice_margin.py]: Why adding `format`?

**Review by manuelcalerosolis — APPROVED** [review]

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
See my previous comment

**Review by pedrobaeza — APPROVED** [review]

File: account_invoice_margin/tests/test_account_invoice_margin.py (hunk 1 above)
Comment (nedaszilinskas) [reply account_invoice_margin/tests/test_account_invoice_margin.py]: @pedrobaeza indeed... I have no clue how that appeared there. Must have been added by a pre-commit formatting (which also sounds unlikely). I removed it now.

**Pull request merged**